save-plugins/fix-plugin/src/commonMain/kotlin/org/cqfn/save/plugins/fix/FixPlugin.kt

```diff
@@ -54,7 +54,7 @@ class FixPlugin(
         return files.chunked(fixPluginConfig.batchSize!!).map { chunk ->
             val pathMap = chunk.map { it.first() to it.last() }
             val pathCopyMap = pathMap.map { (expected, test) -> expected to createTestFile(test) }
-            val testCopyNames = pathCopyMap.joinToString { (_, testCopy) -> testCopy.toString() }
+            val testCopyNames = pathCopyMap.joinToString(separator = fixPluginConfig.separator!!) { (_, testCopy) -> testCopy.toString() }
```

Comment:
Is this function also called if `batchSize == 1`? If so, we should correctly handle nullable separator in that case

Reply:
Despite being nullable, it always has default value, so no need to check additionally